__tests__/app.test.js

```diff
@@ -168,6 +142,59 @@ describe("GET /api/articles", ()=> {
           })
       })
   })
+  test("should sort by query", () => {
+    return request(app)
+    .get("/api/articles")
+    .query({
+        sort_by: "votes",
+        order: "asc"
+    })
+    .expect(200)
+    .then(({body}) => {
+        const articles = body;
+        for (i = 1; i = articles.lenght; i++){
+          expect(articles[i].votes >= articles[i-1].votes)      
```

Comment:
Can you think of a nice lovely package that Paul Copley is responsible for that would allow us to check how things have been sorted